modules/distr-tracing-tempo-install-tempomonolithic-cli.adoc

```diff
@@ -0,0 +1,116 @@
+
+You can install a TempoMonolithic instance from the command line.
+
+.Prerequisites
+
+* An active {oc-first} session by a cluster administrator with the `cluster-admin` role.
++
+[TIP]
+====
+* Ensure that your {oc-first} version is up to date and matches your {product-title} version.
+
+* Run `oc login`:
++
+[source,terminal]
+----
+$ oc login --username=<your_username>
+----
+====
+
+.Procedure
+
+. Run the following command to create a project of your choice for the TempoMonolithic instance that you will create in a subsequent step:
++
+[source,terminal]
+----
+$ oc apply -f - << EOF
+apiVersion: project.openshift.io/v1
+kind: Project
+metadata:
+  name: <project_of_tempomonolithic_instance>
+EOF
+----
+
+. Decide which type of supported storage to use for storing traces: in-memory storage, a Persistent Volume, or object storage.
++
+[IMPORTANT]
+====
+Object storage is not included with the {TempoShortName} and requires setting up an object store by a supported provider: link:https://www.redhat.com/en/technologies/cloud-computing/openshift-data-foundation[Red Hat OpenShift Data Foundation], link:https://min.io/[MinIO], link:https://aws.amazon.com/s3/[Amazon S3], link:https://azure.microsoft.com/en-us/products/storage/blobs/[Azure Blob Storage], or link:https://cloud.google.com/storage/[Google Cloud Storage].
+
+Additionally, opting for object storage requires creating a secret for your object storage bucket in the project that you created for the TempoMonolithic instance. You can do this by running the following command:
+
+[source,terminal]
+----
+$ oc apply -f - << EOF
+<object_storage_secret>
+EOF
+----
+
+For more information, see "Object storage setup".
+
+--
+include::snippets/distr-tracing-tempo-secret-example.adoc[]
+--
+
+====
+
+. Create a TempoMonolithic instance in the project that you created for it.
++
+[TIP]
+====
+You can create multiple TempoMonolithic instances in separate projects on the same cluster.
+====
+
+.. Customize the `TempoMonolithic` custom resource (CR).
++
+include::snippets/distr-tracing-tempo-tempomonolithic-custom-resource.adoc[]
+
+.. Apply the customized CR by running the following command.
++
+[source,terminal]
+----
+$ oc apply -f - << EOF
+<TempoMonolithic_custom_resource>
+EOF
+----
+
+.Verification
+
+. Verify that the `status` of all TempoMonolithic `components` is `Running` and the `conditions` are `type: Ready` by running the following command:
++
+[source,terminal]
+----
+$ oc get tempomonolithic.tempo.grafana.com <metadata_name_of_TempoMonolithic_CR> -o yaml
+----
+
+. Run the following command to verify that the pod of the TempoMonolithic instance is running:
++
+[source,terminal]
+----
+$ oc get pods
+----
+
+. Access the Jaeger UI:
+
+.. Query the route details for the `tempo-<metadata_name_of_TempoMonolithic_CR>-jaegerui` route by running the following command:
++
+[source,terminal]
+----
+$ oc get route
+----
+
+.. Open `\https://<route_from_previous_step>` in a web browser.
```

Comment:
[ISG] Colon leading into substeps:
```suggestion
.. Open `\https://<route_from_previous_step>` in a web browser:
```

Reply:
There are no more substeps. This is the last substep.

Reply:
Oops! Yes you're right, sorry. :]